tests/e2e/network/ensureloadbalancer.go

```diff
@@ -79,14 +80,14 @@ var _ = Describe("Ensure LoadBalancer", func() {
 		tc, err = utils.CreateAzureTestClient()
 		Expect(err).NotTo(HaveOccurred())
 
-		utils.Logf("Creating deployment " + testServiceName)
-		deployment := createNginxDeploymentManifest(testServiceName, labels)
-		_, err = cs.AppsV1().Deployments(ns.Name).Create(context.TODO(), deployment, metav1.CreateOptions{})
+		// Create host exec Pod
+		result, err := utils.CreateHostExecPod(cs, ns.Name, utils.ExecAgnhostPod)
```

Comment:
Can we put this into ValidateInternalServiceConnectivity if it is the only place that needs it?

Reply:
Good idea. Updated.